test/integration/ctl/ctl_create_cr_test.go

```diff
@@ -171,15 +195,20 @@ func TestCtlCreateCR(t *testing.T) {
 				}
 			}
 
+			// Try to set Ready Condition if needed, otherwise the test just times out
+			if test.fetchCert {
+				go setCRReadyCondition(t, cmCl, test.inputArgs[0], test.inputNamespace)
```

Comment:
This seems like a good way to do this, but could we a) move this to use the `wait` package and b) in-line it?

Once we remove the retrying logic, I think this will be a fairly simple bit of code and it'll be a lot more obvious why we're doing this 😄